beaconChain/constants/enums.ts

```diff
@@ -1,24 +1,17 @@
-// https://github.com/ethereum/eth2.0-specs/blob/master/specs/core/0_beacon-chain.md#constants
-
 enum StatusFlags {
```

Comment:
Consider renaming enums to the singular form. (eg: it makes more sense to say a function may return a StatusFlag, rather than a StatusFlags)